src/js/PhotoSphereViewer.js

```diff
@@ -157,7 +157,12 @@ function PhotoSphereViewer(options) {
       cropped_height: 0,
       cropped_x: 0,
       cropped_y: 0
-    }
+    },
+    usexmpdata: false,
```

Comment:
same remark as your previous PR : what is this ? aren't you mixing "prop" and "config"